nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -982,6 +982,12 @@ services.transmission.settings.rpc-bind-address = "0.0.0.0";
       the previous behaviour using <literal>undervolt.useTimer</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Following its deprecation in 20.03, the Perl NixOS test driver has been removed.
+     All remaining tests have been ported to the Python test framework.
+    </para>
+   </listitem>
```

Comment:
I think it might be better if this was more prominent as a highlight and mention that anyone that was using the old `testing.nix` has to use `testing-python.nix` as a part of upgrading to 20.09

Reply:
Updated.